biz.aQute.bndlib/src/aQute/bnd/build/BndProgress.java

```diff
@@ -0,0 +1,32 @@
+package aQute.bnd.build;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import aQute.bnd.service.progress.ProgressPlugin;
+
+public class BndProgress implements ProgressPlugin {
```

Comment:
I keep thinking about the class name. Perhaps `LoggingProgressPlugin` is more appropriate?